x-pack/plugins/secops/public/components/timeline/body/sort/index.ts

```diff
@@ -4,7 +4,7 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { Direction } from '../../../../graphql/types';
+import { SortDirection as Direction } from '../../../../graphql/types';
```

Comment:
Would it make sense to refactor to use [/@elastic/eui/src/services/sort/sort_direction.ts](https://github.com/elastic/eui/blob/121101f834b274dcbe4951ed7aa5ab0c5cd8c9c7/src/services/sort/sort_direction.ts)'s `SortDirection` type as the shared GraphQL type is in reference to Elasticsearch's sort direction values? They're the same values for what it's worth, but contextually one is UI sorting and the other is query sorting. This file also exports it's own `SortDirection` type, so that's why you'll see the `import { SortDirection as Direction }` throughout these timeline files. Happy to rename this exported `SortDirection` (line 11 below) to `TimelineSortDirection` or something else if that makes more sense.